main/supervisor/APKBUILD

```
@@ -1,14 +1,15 @@
 # Contributor: Peter Bui <pnutzh4x0r@gmail.com>
 # Maintainer:
 pkgname=supervisor
-pkgver=3.3.5
+pkgver=4.0.3
 pkgrel=0
 pkgdesc="A system for controlling process state under UNIX"
 url="http://supervisord.org"
 arch="noarch"
 license="BSD"
```

```suggestion
license="custom"
```

This is only inspired by `BSD-4-Clause` but diverged significantly. Please be sure to include both `COPYRIGHT.txt` and `LICENSES.txt` in the package.

---

Fixed as suggested